src/server/auth/server/api_server.go

```diff
@@ -92,7 +92,8 @@ const (
 	SamlPort = 654
 )
 
-var defaultAuthConfig = auth.AuthConfig{
```

Comment:
Auth tests were moved into a subpackage to break a circular dependency between `auth/server` and `testutil` (same for `pfs/server`).  Some package-local variables were changed to public for access by tests.